docs/getting_started/index.rst

```diff
@@ -46,7 +46,7 @@ integration of the package or the ``Standalone`` version.
 
              composer require schranz-search/spiral-bridge
 
-| The project provides adapters to different search engines, atleast one is required.
+| The project provides adapters to different search services, atleast one is required.
```

Comment:
The problem is that various providers call their products "search engines" themselves:

> Meilisearch is a flexible and powerful user-focused search engine…

https://www.meilisearch.com

> Elasticsearch is a distributed, RESTful search and analytics engine…

https://www.elastic.co/elasticsearch/

> Free and Open, Distributed, RESTful Search Engine

https://github.com/elastic/elasticsearch

> Typesense is a modern, privacy-friendly, open source search engine…

https://typesense.org

And also Wikipedia:

> Elasticsearch is a search engine…

https://en.wikipedia.org/wiki/Elasticsearch

---

But Wikipedia uses another term: [Search engine software](https://en.wikipedia.org/wiki/List_of_search_engine_software)

Reply:
Maybe adding some `abbr` can avoid confusing between terms: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-abbr

Reply:
I would prefer unambiguous and clearly readable names. Maybe something like these:

- Search engine software
- Search client
- …

vs.

- SEAL engine
- Abstract engine
- Search connector
- …

If the engine class is based on a design pattern, then this name could also be used / added.

:thinking:

Reply:
Didn't wanted to use `client` as client is mostly provided by specific adapter classes and could be confused with that one easily, so in the PHP autocomplete there is not much conflicting with the Engine class.